pkg/nsx/services/vpc/vpc.go

```diff
@@ -568,8 +580,31 @@ func (s *VPCService) CreateOrUpdateVPC(obj *v1alpha1.NetworkInfo) (*model.Vpc, *
 		return existingVPC[0], &nc, nil
 	}
 
+	// build NSX LBS
+	var createdLBS *model.LBService
+	// TODO(gran) use switch to enable/disable NSXLB and AVI
+	if s.NSXConfig.NsxConfig.NSXLBEnabled() {
+		lbsSize := s.NSXConfig.NsxConfig.GetNSXLBSize()
+		// TODO(gran) Uncomment after #628 merged
+		//if nc.LbServiceSize != "" {
+		//	lbsSize = nc.LbServiceSize
+		//}
```

Comment:
This can be removed, we would not expose LB size in NetworkConfiguration CR, will use SMALL by default

Reply:
Fixed.